plugins/kubescape.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kubescape
+spec:
+  homepage: https://kubescape.io/
+  shortDescription: An open-source Kubernetes security platform for your IDE, CI/CD pipelines, and clusters
```

Comment:
Try to be more concise as this will be truncated if too long. Try to remove words that don't add value when you have limited space:
e.g.

> Scan resources and cluster configs for seucirty